### ISSUE TITLE
allow_ajax doesn't work with django-debug-toolbar==1.9.1

### ISSUE THREAD

**Issue by benjaminrigaud** [djsutho/django-debug-toolbar-request-history#16]
The SQL tab shows a 404: NOT FOUND
And the Request History tab shows no ajax requests.

It used to work fine with ddt==1.9.0

The doc says `For vesions of the debug toolbar less than 1.8 you can record ajax requests by adding SHOW_TOOLBAR_CALLBACK to DEBUG_TOOLBAR_CONFIG`

Does it mean we shouldn't be able to see Ajax requests with DDT >= 1.8.0?

Ben

**Comment by djsutho** [comment]
Thanks for the report. I'll try to look into this on the weekend. With versions < 1.8 the `SHOW_TOOLBAR_CALLBACK` is required, versions above that should set this up automatically to record all non-toolbar related requests.

**Comment by benjaminrigaud** [comment]
Thank you.

In Django 1.8 I have to configure SHOW_TOOLBAR_CALLBACK to see the ajax requests: it is breaking everything since DDT 1.9.1 :)

Tell me if you need help to reproduce / debug anything.

**Comment by djsutho** [comment]
@benjaminrigaud Can I get you to install the master version with:
```bash
pip install git+https://github.com/djsutho/django-debug-toolbar-request-history.git
```
and re-test this?

**Comment by benjaminrigaud** [comment]
@djsutho Sure. I'll do that now.

**Comment by benjaminrigaud** [comment]
@djsutho it works great! Thank you.

**Comment by djsutho** [comment]
This has now been released to pypi as version 0.0.6
